### ISSUE TITLE
cleanup(menu-items): remove menu items module

### ISSUE THREAD

**Issue by AmitMY** [AnSyn/ansyn#640]
**I'm submitting a ...**  (check one with "x")

```
[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request
[x] Task/UserStory
```

**Description:**
No one is using `menu-items.module`, we can just remove it.

**Comment by yairtawil** [comment]
resolved via #647

**Comment by yairtawil** [comment]
resolved via #647
